pdcupdater/utils.py

```diff
@@ -497,6 +497,15 @@ def interesting_container_tags():
     #return ['%s-docker' % tag for tag in tags]
 
 
+def all_tags_from_pdc(pdc):
+    for release in pdc.get_paged(pdc['releases']._):
+        if not release['active']:
```

Comment:
Can you do this filter on the API? I know you can in the URL by:
https://pdc.engineering.redhat.com/rest_api/v1/releases/?active=True